react/__tests__/Price.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { mountWithIntl, loadTranslation } from 'enzyme-react-intl'
+
+import { getIntlContextInfo, getIntlInstance } from './helpers/intlHelper'
+import Price from '../Price'
+
+describe('<Price /> component', () => {
+  let renderComponent, productPropsMock, configuration
+  let intl, currencyOptions
+
+  beforeEach(() => {
+    const { context, childContextTypes, locale } = getIntlContextInfo()
+
+    currencyOptions = {
+      style: 'currency',
+      currency: context.culture.currency,
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2,
+    }
+
+    productPropsMock = {
+      listPrice: 200,
+      sellingPrice: 170,
+      installments: 3,
+      installmentPrice: 50,
+    }
+    configuration = {}
```

Comment:
Don't rely on global variables to pass props to the render function, as it makes the code harder to read. A better alternative would be to just add an argument to the `renderComponent` function like this
```jsx
const renderComponent = (props = {}) => mountWithIntl(...)
```
Also, you don't need to redefine this function before each test case, you can put the implementation outside the `beforeEach` function callback.